MathLib/KelvinVector-impl.h

```diff
@@ -20,6 +22,13 @@ double Invariants<KelvinVectorSize>::equivalentStress(
     return std::sqrt(3 * J2(deviatoric_v));
 }
 
+template <int KelvinVectorSize>
+double Invariants<KelvinVectorSize>::Norm(
```

Comment:
That's the Frobenius norm of the tensor represented by `deviatoric_v`, isn't it?
I'd call the function `FrobeniusNorm()`. And it's also valid for general Kelvin vectors, isn't it?

Reply:
Yes, it it. Changed accordingly.